acceptance-tests/src/main/java/io/blueocean/ath/LocalDriver.java

```diff
@@ -1,20 +1,23 @@
 package io.blueocean.ath;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.FluentWait;
 
 import java.util.List;
-import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
 
 /**
- * Wrapper around an underlying WebDriver that
- * consistently handles waits automatically.
- *
- * Accepts expressions for css and xpath, if the provided lookup starts with a /, XPath is used
+ * Provides access to the current WebDriver instance.
+ * Contains some utility methods for working with waits and SmartWebElement.
+ * TODO: rename this class as it's *not* a WebDriver but rather more of a DriverHolder.
```

Comment:
All of the client code was calling `LocalDriver.getDriver()` which returned the thread local'd `RemoteWebDriver`. No need for `LocalDriver` to impl `WebDriver` as far as I can see.